components/prism-io.js

```diff
@@ -0,0 +1,31 @@
+Prism.languages.io = {
+    'comment': [
+        {
+            pattern: /(^|[^\\])\/\*[\s\S]*?(?:\*\/|$)/,
+            lookbehind: true
+        },
+        {
+            pattern: /(^|[^\\:])\/\/.*/,
+            lookbehind: true
+        },
+        {
+            pattern: /(^|[^\\:])#.*/,
```

Comment:
I think you can remove the `:` in the lookbehind as it is (was?) used to prevent URLs to match `//` comments (not even sure it's useful anymore).